pytensor/link/pytorch/dispatch/elemwise.py

```diff
@@ -37,6 +38,64 @@ def dimshuffle(x):
     return dimshuffle
 
 
+@pytorch_funcify.register(Sum)
+def pytorch_funcify_sum(op, **kwargs):
+    def torch_sum(x):
+        return torch.sum(x, dim=op.axis)
+
+    return torch_sum
+
+
+@pytorch_funcify.register(All)
+def pytorch_funcify_all(op, **kwargs):
+    dim = op.axis
+
+    def torch_all(x):
+        return torch.all(x, dim=dim)
+
+    return torch_all
+
+
+@pytorch_funcify.register(Prod)
+def pytorch_funcify_prod(op, **kwargs):
+    dim = op.axis[0]
```

Comment:
Why axis[0]?

Reply:
`op.axis` is a tuple, pytorch expects integers for `dim`

Reply:
Then we need to change the logic, because it is possible for them to be tuples with more than one entry

Reply:
We could do something like this:
```
for d in op.axis:
     x=torch.prod(x, dim=d, keepdim=True) # To make sure constant shape, we can reshape in the end
                
for _ in op.axis:
     x.squeeze_(-1)
```

Reply:
If you reduce in reversed order you don't have to worry about the keepdims. Sounds good, a bit surprising that they don't support multiple axes